jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -616,12 +626,60 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
                 CodetransformTelemetry.totalRunTime(
                     codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
                     codeTransformResultStatusMessage = "JobCancelled",
-                    codeTransformRunTimeLatency = calculateTotalLatency(CodeTransformTelemetryState.instance.getStartTime(), Instant.now())
+                    codeTransformRunTimeLatency = calculateTotalLatency(CodeTransformTelemetryState.instance.getStartTime(), Instant.now()),
+                    codeTransformLocalJavaVersion = getJavaVersionFromProjectSetting(),
+                    codeTransformLocalMavenVersion = getMavenVersions(),
                 )
             }
         }
     }
 
+    private fun getJavaVersionFromProjectSetting(): String? = project.tryGetJdk()?.toString()
```

Comment:
Question: Don't we already have a reference to function getting this java version from somewhere?

Reply:
I think `Utils.kt` in the codemodernizer folder would be a good place.

Reply:
Good call, can move it to util.